changes/next/imap_jmapaccess

```diff
@@ -0,0 +1,19 @@
+
+Description:
+
+Adds support for IMAP JMAPACCESS response code (draft-ietf-extra-jmapaccess).
```

Comment:
Unless this gets tagged as `Do Not Merge` until that I-D draft is a standard, should this change log rather call out that this is a prototype implementation?

Reply:
Switched PR to draft until IETF 199